shared/desktop/renderer/remote-tracker.js

```diff
@@ -30,6 +32,7 @@ class RemoteTracker extends Component<void, Props, void> {
       <div>
         {Object.keys(trackers)
           .filter(username => !trackers[username].closed)
+          .slice(0, MAX_TRACKERS)
```

Comment:
I think you want the list to remain the same so as new ones come they don't disappear and thrash out